apps/studio/schemas/testimonial.ts

```diff
@@ -35,7 +35,7 @@ export default defineType({
       title: "Rating 0-5",
       type: "number",
       initialValue: 5,
-      validation: (Rule) => Rule.min(0).max(5).integer(),
+      validation: (Rule) => Rule.min(0).max(5).integer().required(),
```

Comment:
I can't remember if this was needed, I might have added it just to test

Reply:
It is needed or else the generated schema will expect either number or null, instead of just number. By limiting to just numbers, we are able to benefit from the strong typing which simplifies implementation.